Dirty fix, to make dvdauthor mode work, until I find out, how to link
ffmpeg libs directly
--- a/src/import/k9avidecode.cpp
+++ b/src/import/k9avidecode.cpp
@@ -40,10 +40,10 @@
     SwscaleHandle=0;
     glibref=0;
     if (glibref==0) {
-        CodecHandle=dlopen("libavcodec.so",RTLD_LAZY ); //| RTLD_GLOBAL
-        FormatHandle=dlopen("libavformat.so",RTLD_LAZY);
+        CodecHandle=dlopen("libavcodec.so.52",RTLD_LAZY ); //| RTLD_GLOBAL
+        FormatHandle=dlopen("libavformat.so.52",RTLD_LAZY);
 #ifdef HAVE_SWSCALE
-	SwscaleHandle=dlopen("libswscale.so",RTLD_LAZY);
+	SwscaleHandle=dlopen("libswscale.so.0",RTLD_LAZY);
 #endif
     }
     m_error="";
